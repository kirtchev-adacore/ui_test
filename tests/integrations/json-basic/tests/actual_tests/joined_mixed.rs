#![deny(unused_mut, unused_variables)]

fn main() {
    let mut x = 0u32;
    //~^ unused_variables
    //~| unused_mut

    //~| unused_mut
    //~v unused_variables
    let mut y = 0u32;
}
