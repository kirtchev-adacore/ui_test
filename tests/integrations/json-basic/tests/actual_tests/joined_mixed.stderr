error: unused variable: `x`
 --> tests/actual_tests/joined_mixed.rs:4:13
  |
4 |     let mut x = 0u32;
  |             ^ help: if this is intentional, prefix it with an underscore: `_x`
  |
note: the lint level is defined here
 --> tests/actual_tests/joined_mixed.rs:1:21
  |
1 | #![deny(unused_mut, unused_variables)]
  |                     ^^^^^^^^^^^^^^^^

error: unused variable: `y`
  --> tests/actual_tests/joined_mixed.rs:10:13
   |
10 |     let mut y = 0u32;
   |             ^ help: if this is intentional, prefix it with an underscore: `_y`

error: variable does not need to be mutable
 --> tests/actual_tests/joined_mixed.rs:4:9
  |
4 |     let mut x = 0u32;
  |         ----^
  |         |
  |         help: remove this `mut`
  |
note: the lint level is defined here
 --> tests/actual_tests/joined_mixed.rs:1:9
  |
1 | #![deny(unused_mut, unused_variables)]
  |         ^^^^^^^^^^

error: variable does not need to be mutable
  --> tests/actual_tests/joined_mixed.rs:10:9
   |
10 |     let mut y = 0u32;
   |         ----^
   |         |
   |         help: remove this `mut`

error: aborting due to 4 previous errors

